### PR TITLE
CDAP-2220: Merge StructuredRecordToCubeFactTransform and CubeSink

### PR DESCRIPTION
**Pull request by abaranau**

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * the License.
  */
 
-package co.cask.cdap.templates.etl.transforms;
+package co.cask.cdap.templates.etl.batch.sinks;
```
Comment (albertshau) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java]: this probably makes more sense to be in common, that way the real time sink can use it too

**Comment by albertshau** [comment]
1 comment, otherwise lgtm

**Comment by abaranau** [comment]
http://builds.cask.co/browse/CDAP-DUT1581-1

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java (hunk 1 above)
Comment (abaranau) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java]: true!

Comment (abaranau) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/batch/sinks/StructuredRecordToCubeFact.java]: can I move it along with https://issues.cask.co/browse/CDAP-2221 (Implement realtime CubeSink)?

**Comment by gokulavasan** [comment]
@abaranau Start a build under http://builds.cask.co/browse/CDAP-DT since DUT does not include templates profile.

**Comment by abaranau** [comment]
@gokulavasan Thanks! Started! http://builds.cask.co/browse/CDAP-DT50-1

**Pull request merged**